charts/argo-cd/templates/_helpers.tpl

```diff
@@ -188,6 +188,8 @@ app.kubernetes.io/managed-by: {{ .context.Release.Service }}
 app.kubernetes.io/part-of: argocd
 {{- with .context.Values.global.additionalLabels }}
```

Comment:
I don’t see difference from this config that does the same thing.

Reply:
Good catch here. Thanks @pdrastil clothing this PR